### PR TITLE
[PAPP-32437] pastebin: Bugfix - Bug fix for accessing path to the vault temp directory

### PR DESCRIPTION
**Pull request by bbielinski-splunk**
Please ensure your pull request (PR) adheres to the following guidelines:

- Please refer to our contributing documentation for any questions on submitting a pull request, link: [Contribution Guide](https://github.com/splunk-soar-connectors/.github/blob/main/.github/CONTRIBUTING.md)

#### Please check if your PR fulfills the following requirements:
- [x] Testing of all the changes has been performed (for bug fixes / features)
- [x] The manual_readme_content.md has been reviewed and added / updated if needed (for bug fixes / features)
- [x] Use the following format for the PR description: `<App Name>: <PR Type> - <PR Description>`
- [x] Provide release notes as part of the PR submission which describe high level points about the changes for the upcoming GA release.
- [x] Verify all checks are passing.
- [x] Do NOT use the `next` branch of the forked repo. Create separate feature branch for raising the PR.
- [x] Do NOT submit updates to dependencies unless it fixes an issue.
#### Please check the type of change your PR introduces:
- [ ] New App
- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Documentation
- [ ] Other (please describe):

## Security Considerations (REQUIRED)
- [x] If you are exposing any endpoints using a [REST handler](https://docs.splunk.com/Documentation/SOAR/current/DevelopApps/RESTHandlers),
  please document them in the `manual_readme_content.md`.
- [x] If this is a new connector or you are adding new actions
    - [x] Please document in the `manual_readme_content.md` all methods (eg, OAuth) used to authenticate
      with the service that the connector is integrating with.
    - [x] If any actions are unable to run on SOAR Cloud, please document this in the `manual_readme_content.md`.
- [x] Are you introducing any new cryptography modules? If yes, please elaborate their purpose:
- [x] Are you are accessing the file system? If yes, please verify that you are only accessing paths returned through
the [Vault](https://docs.splunk.com/Documentation/SOARonprem/5.2.1/DevelopApps/AppDevAPIRef#Vault) API.
- [x] Are you are marking code to be ignored by Semgrep with [`nosemgrep`](https://semgrep.dev/docs/ignoring-files-folders-code/#ignoring-code-through-nosemgrep)?
If yes, please provide justification in an additional comment next to the ignored code.

## Release Notes (REQUIRED)
- Bug fix for accessing path to the vault temp directory [PAPP-32437]

## What is the current behavior? (OPTIONAL)
n/a

## What is the new behavior? (OPTIONAL)
n/a

## Other information (OPTIONAL)
n/a

## Pay close attention to (OPTIONAL)
n/a

## Screenshots (if relevant)
n/a

---
Thanks for contributing!

### CODE REVIEW

File: pastebin_connector.py [hunk 1]
```diff
@@ -65,12 +65,8 @@ def _convert_to_client_tz(self, t_str):
...
 
-        if hasattr(Vault, 'get_vault_tmp_dir'):
-            temp_dir = Vault.get_vault_tmp_dir()
-        else:
-            temp_dir = "opt/phantom/vault/tmp"
-
-        temp_dir = temp_dir + '/{}'.format(uuid.uuid4())
+        vault_tmp_dir = Vault.get_vault_tmp_dir()
+        temp_dir = vault_tmp_dir + '/{}'.format(uuid.uuid4())
         os.makedirs(temp_dir)
```
Comment (bb-splunk) [inline pastebin_connector.py]: Maybe this whole thing could be done more like this https://github.com/splunk-soar-connectors/microsoftwindowsremotemanagement/pull/22/files#diff-ca6e181c530d0fa3e279b1e9396a120bdd444ea4d52c1e5043f7cfa11911249dR455? Can you check?

File: README.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ Connector Version: 2.0.2
 Product Vendor: PasteBin  
 Product Name: PasteBin  
 Product Version Supported (regex): ".\*"  
-Minimum Product Version: 5.3.3  
+Minimum Product Version: 5.5.0
```
Comment (mposluszny-splunk) [inline README.md]: Are we sure that this is min prod version now?

Comment (bbielinski-splunk) [reply README.md]: Good catch, static tests were run before latest changes to min ver were introduced, bumped to 6.1.1

File: pastebin_connector.py (hunk 1 above)
Comment (bbielinski-splunk) [reply pastebin_connector.py]: It can be done this way, applied.

File: pastebin.json [hunk 3]
```diff
@@ -621,7 +621,7 @@
         }
     ],
     "pip_dependencies": {
-        "wheel": [
+        "pypi": [
```
Comment (bb-splunk) [inline pastebin.json]: This should be reverted before merge :D

Comment (bb-splunk) [reply pastebin.json]: Same below

Comment (bbielinski-splunk) [reply pastebin.json]: Right, fixed

File: pastebin_connector.py [hunk 4]
```diff
@@ -63,39 +60,18 @@ def _convert_to_client_tz(self, t_str):
...
-                if not success:
-                    return action_result.set_status(phantom.APP_ERROR, message)
-                action_result.add_data({"vault_id": attachment_vault_id})
+            resp = Vault.create_attachment(smart_str(file_data), container_id, file_name=file_name)
+        except Exception as ex:
+            return action_result.set_status(
+                phantom.APP_ERROR,
+                f"Unable to add file to the vault for attachment name: "
+                f"{file_name}. Error: "
+                f"{self._get_error_message_from_exception(ex)}"
```
Comment (bb-splunk) [inline pastebin_connector.py:70]:
```suggestion
                f"Unable to add file to the vault for attachment name: {file_name}. "
                f"Error: {self._get_error_message_from_exception(ex)}"
```

**Review by bb-splunk — APPROVED** [review]

File: pastebin_connector.py (hunk 4 above)
Comment (bbielinski-splunk) [reply pastebin_connector.py:70]: Unfortunately can't apply. The lines will exceed the 79 character limit.

**Review by mposluszny-splunk — APPROVED** [review]

**Review by alexa-phantom — APPROVED** [review]

**Pull request merged**